cypress/integration/v2/features/dynamic-content/site-data.spec.js

```diff
@@ -0,0 +1,132 @@
+
+/**
+ * External dependencies
+ */
+import { registerTests } from '~stackable-e2e/helpers'
+
+describe( 'Dynamic Content - Site', registerTests( [
+	matchSiteData,
+	adjustFieldOptions,
+	adjustFieldValues,
+] ) )
+
+const fields = [ 'Site Title', 'Site Tagline', 'Site URL' ]
+const testString = [ 'Test Site Title', 'Test Site Tagline', 'Test Site URL' ]
+
+const adjustSiteField = ( fieldName, fieldOptions = {} ) => {
+	 cy.adjustDynamicContent( 'ugb/cta', 0, '.ugb-cta__title', {
+		 source: 'Site',
+		 fieldName,
+		 fieldOptions,
+	 } )
+}
+
+function matchSiteData() {
+	 it( 'should match dynamic content in site fields', () => {
+		 cy.setupWP()
+		 Object.keys( fields ).forEach( fieldName => {
+			 cy.newPost()
+			 cy.addBlock( 'ugb/cta' )
+			 cy.typePostTitle( `${ fields[ fieldName ] } Test` )
```

Comment:
we might not need to add this command in the test since there's no purpose adding title to the current post.